src/Symfony/Component/Notifier/Bridge/Discord/DiscordOptions.php

```diff
@@ -27,12 +28,22 @@ public function __construct(
 
     public function toArray(): array
     {
-        return $this->options;
+        $options = $this->options;
+        unset($options['recipient_id']);
```

Comment:
if the recipient id is not meant to be part of the array of options, I would suggest storing it as a separate property instead.

Reply:
I've done it the same way as in slack transport

Reply:
Updated